dflib/src/main/java/org/dflib/exp/bool/BoolScalarExp.java

```diff
@@ -15,6 +15,9 @@ public class BoolScalarExp extends ScalarExp<Boolean> implements Condition {
 
     private final boolean boolValue;
 
```

Comment:
Let's replace variable access with a static method. E.g. take a look at `CountExp`. So maybe smth like `Condition getTrue()` and `Condition getFalse()`